tests/integration/test_fim/test_files/test_follow_symbolic_link/common.py

```diff
@@ -4,8 +4,8 @@
 import subprocess
 import sys
 
-from wazuh_testing.fim import callback_audit_loaded_rule, create_file, REGULAR, SYMLINK, callback_symlink_scan_ended, \
-    change_internal_options
+from wazuh_testing.fim import callback_end_audit_reload_rules, create_file, REGULAR, SYMLINK, \
+    callback_symlink_scan_ended, change_internal_options
```

Comment:
You can apply the same change to the imports here too.

Reply:
I have removed an unused import and now all the imports are in one line. 
I think that now this change doesn't improve the legibility.